### PR TITLE
Added verify and commit screen

### LINKED ISSUE #1591: Workflows-Verify and Commit Screen dev.
Verify and Commit Screen Development for Litmus Portal Frontend.

### PR DESCRIPTION
**Pull request by Saranya-jena**
Signed-off-by: Saranya-jena <saranyajena29@yahoo.com>

**What this PR does / why we need it**:
This PR includes 'Verify and Commit' screen including some other components.
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #1591 

**Checklist**

* [x] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [ ] Have you added debug messages where necessary? 
* [x] Have you added task comments where necessary? 
* [ ] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

File: litmus-portal/frontend/src/components/ScheduleWorkflow/styles.ts [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    fontSize: '1.5625rem',
+  },
+  schBody: {
+    width: '32.18rem',
+    height: '3.375',
+  },
+  captionText: {
+    marginLeft: theme.spacing(3.75),
+    fontSize: '0.75rem',
+
```
Comment (rajdas98) [inline litmus-portal/frontend/src/components/ScheduleWorkflow/styles.ts:35]: remove this

File: litmus-portal/frontend/src/components/CustomStepper/index.tsx [hunk 2]
```diff
@@ -11,6 +11,8 @@ import FinishModal from '../FinishModal';
 import Loader from '../Loader';
 import ReliablityScore from '../ReliabilityScore';
 import WorkflowCluster from '../WorkflowCluster';
+import ScheduleWorkflow from '../ScheduleWorkflow';
+import VerifyCommit from '../Verify and Commit';
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/CustomStepper/index.tsx:15]: Change folder name to VerifyCommit

File: litmus-portal/frontend/src/components/CustomText/index.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        disabled={isDisabled}
+        id={id}
+        defaultValue={value}
+        multiline
+        rowsMax={6}
+        InputProps={{
+          disableUnderline: true,
+          style: {
+            width,
+            color: 'rgba(0,0,0)',
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/CustomText/index.tsx:34]: move this to styles.ts file and take color from theme data

File: litmus-portal/frontend/src/components/CustomTime/index.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        style={{
+          width: '5.375rem',
+          height: '2.75rem',
+          margin: 10,
+        }}
+        inputVariant="outlined"
+        inputProps={{
+          style: {
+            fontSize: '0.75rem',
+            color: '#000000',
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/CustomTime/index.tsx:34]: color from theme data, no inline styles

File: litmus-portal/frontend/src/components/ResultModal/index.tsx [hunk 5]
```diff
@@ -134,13 +134,17 @@ const ResultModal: React.FC<ResultModalProps> = ({ isOpen, testValue }) => {
                         {row.weight}
                         &nbsp; points
                         <br />
-                        <LinearProgressBar value={row.weight} />
+                        <div style={{ width: 150 }}>
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/ResultModal/index.tsx:137]: width in px value

File: litmus-portal/frontend/src/components/ResultModal/index.tsx [hunk 6]
```diff
@@ -134,13 +134,17 @@ const ResultModal: React.FC<ResultModalProps> = ({ isOpen, testValue }) => {
...
+                          <LinearProgressBar value={row.weight} />
+                        </div>
                       </TableCell>
                       <TableCell align="center" className={classes.tablePoints}>
                         {row.points}
                         &nbsp; points
                         <br />
-                        <LinearProgressBar value={row.points} />
+                        <div style={{ width: 150 }}>
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/ResultModal/index.tsx:145]: width in px values

File: litmus-portal/frontend/src/components/ScheduleWorkflow/index.tsx [hunk 7]
```diff
@@ -0,0 +1,315 @@
...
+  Button,
+  RadioGroup,
+  FormControlLabel,
+  Radio,
+} from '@material-ui/core';
+
+import ToggleButtonGroup from '@material-ui/lab/ToggleButtonGroup/ToggleButtonGroup';
+import TextField from '@material-ui/core/TextField/TextField';
+import useStyles from './styles';
+import calendar from '../../icons/calendar.png';
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/ScheduleWorkflow/index.tsx:15]: images should be in the public folder

File: litmus-portal/frontend/src/components/SideBar/styles.ts [hunk 8]
```diff
@@ -18,28 +16,25 @@ const useStyles = makeStyles((theme: Theme) => ({
...
-    width: 65,
-    height: 30,
-    fontWeight: 'normal',
-    fontFamily: 'Ubuntu Condensed',
-    fontStyle: 'normal',
-    fontSize: 26,
-    left: 81,
-    top: 24,
+    width: '4.0625rem',
+    height: '1.875',
```
Comment (gdsoumya) [inline litmus-portal/frontend/src/components/SideBar/styles.ts:23]: No specific unit used

File: litmus-portal/frontend/src/components/Verify and Commit/styles.ts [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+  },
+  description: {
+    width: '32.18rem',
+    marginTop: theme.spacing(3.25),
+    marginBottom: theme.spacing(7.5),
+    fontSize: '1.0625rem',
+  },
+  suBody: {
+    width: '32.18rem',
+    height: '3.375',
```
Comment (gdsoumya) [inline litmus-portal/frontend/src/components/Verify and Commit/styles.ts:38]: Same here

**Pull request merged**